algorithms/tamil.sbl

```diff
@@ -68,7 +68,7 @@ stringdef vs_au    '{U+0BCC}'
 /* Pulli */
 stringdef pulli    '{U+0BCD}'
 
-/* AU length markk */
+/* AU length mark */
 stringdef au_lmark '{U+0BD7}'
```

Comment:
I'm not at all certain about this, but https://www.compart.com/en/unicode/U+0BD7 doesn't conflict with this change.